src/client/components/DateDivider.tsx

```diff
@@ -27,25 +27,31 @@ export function DateDivider({ timestamp }: { timestamp: Date }) {
 }
 
 const Root = styled.div({
-  width: '100%',
-  position: 'relative',
-  padding: '20px 0',
+  '&&': {
```

Comment:
(newbie styled components question) why do we need these?

Reply:
It might not be the best way to do it, but it's the only way I found. So these components already have these properties set in the css coming from the sdk. If we don't put the &&, the styles will have lower specificity. This is a way of "bumping" the styles to the top.

Reply:
This is without:
<img width="441" alt="Screenshot 2024-05-17 at 09 05 49" src="https://github.com/getcord/clack/assets/102948032/32ac96bb-b08c-420e-9d11-c66433ccda58">
<img width="436" alt="Screenshot 2024-05-17 at 09 05 32" src="https://github.com/getcord/clack/assets/102948032/37d89b4e-1adf-41c3-8ace-e156ae5a55c6">


This is with:
<img width="444" alt="Screenshot 2024-05-17 at 09 06 24" src="https://github.com/getcord/clack/assets/102948032/e3a5da68-1f73-4445-a522-2d32306b3d2f">
<img width="439" alt="Screenshot 2024-05-17 at 09 06 17" src="https://github.com/getcord/clack/assets/102948032/364e8733-9ff4-4d84-b947-5c09db274972">

Reply:
Thanks for explaining! 😄 

Oh yeah. Our reset is 0,1,0 -- the idea is to use `.cord-v2` along with whatever thing you're selecting to have at least 0,2,0 specificity.